src/roiextractors/extractors/tiffimagingextractors/scanimagetiffimagingextractor.py

```diff
@@ -71,6 +74,9 @@ def __init__(
         self._num_planes = imaging_extractors[0].get_num_planes()
         super().__init__(imaging_extractors=imaging_extractors)
 
+    def get_sampling_frequency(self) -> float:
+        return self._imaging_extractors[0].get_sampling_frequency()
+
```

Comment:
This is already defined in MultiImagingExtractor

Reply:
Yes, that's a mistake.